app/pages/resource/reservation-form/catering/catering-form/CateringFormContainer.js

```diff
@@ -0,0 +1,176 @@
+import isEqual from 'lodash/isEqual';
+import values from 'lodash/values';
+import moment from 'moment';
+import React, { Component, PropTypes } from 'react';
+import Button from 'react-bootstrap/lib/Button';
+import Col from 'react-bootstrap/lib/Col';
+import ControlLabel from 'react-bootstrap/lib/ControlLabel';
+import FormControl from 'react-bootstrap/lib/FormControl';
+import FormGroup from 'react-bootstrap/lib/FormGroup';
+import Glyphicon from 'react-bootstrap/lib/Glyphicon';
+import InputGroup from 'react-bootstrap/lib/InputGroup';
+import Row from 'react-bootstrap/lib/Row';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+
+import uiActions from 'actions/uiActions';
+import CateringMenuItems from './CateringMenuItems';
+import CateringOrderTable from '../CateringOrderTable';
+import cateringUtils from '../utils';
+
+const defaultCateringTimeSelector = (state) => {
+  const reservationTimes = state.form.resourceReservation.values.time;
+  return reservationTimes ? moment(reservationTimes.begin).format('HH:mm') : '12:00';
+};
+
+export const selector = createSelector(
+  state => state.catering,
+  state => state.data.cateringMenuItems || {},
+  defaultCateringTimeSelector,
+  state => Number(state.form.resourceReservation.values.numberOfParticipants || 1),
+  (cateringData, cateringMenuItems, defaultCateringTime, defaultItemQuantity) => ({
+    cateringData,
+    cateringMenuItems,
+    defaultCateringTime,
+    defaultItemQuantity,
+  })
+);
+
+class CateringFormContainer extends Component {
+  static propTypes = {
+    cateringData: PropTypes.object.isRequired,
```

Comment:
- [ ] This should be a shape describing the attributes used in this component, like additionalInfo and projectNumber.